service-registry/src/main/java/io/servicecomb/serviceregistry/api/Const.java

```diff
@@ -148,6 +148,10 @@ private Const() {
   public static final String PATH_CHECKSESSION = "checksession";
 
   public static final String URL_PREFIX = "urlPrefix";
-  
+
   public static final String INSTANCE_PUBKEY_PRO = "publickey";
+
+  public static final String GROUPID = "io.servicecomb";
+
+  public static final String ARTIFACTID = "java-chassis";
```

Comment:
are you sure you can read version by this artifactid?

Reply:
when mvn package, the version can be read,